### ISSUE TITLE
Melody search: we may want to prevent user from inputting further notes once they've reached the end of the line

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#1173]
Currently, on the chant search page, clicking on the staff will continue to add notes even if the notes are not visible because the staff has been is truncated. (You can replicate this by making the window narrow, adding a bunch of notes, and then widening the window to reveal notes that previously weren't visible).

It's worth noting that when the window is wide, the tool will stop accepting notes once 14 have been added.

We might consider either
- preventing further notes from being added once the visible part of the staff is full
- shrinking the staff once it is full to allow more notes to be added

It may be that the best solution is to do nothing - I'm not sure what the right course of action is when a wide window with a long query is made more narrow, for example. But this is worth thinking about, at least.
